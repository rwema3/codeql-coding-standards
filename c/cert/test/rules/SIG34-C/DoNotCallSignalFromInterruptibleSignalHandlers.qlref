rules/SIG34-C/DoNotCallSignalFromInterruptibleSignalHandlers.ql