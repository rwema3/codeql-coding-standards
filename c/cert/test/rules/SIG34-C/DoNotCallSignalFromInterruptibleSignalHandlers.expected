| test.c:4:7:4:12 | call to signal | Reasserting handler bindings introduces a race condition on nonpersistent platforms and is redundant otherwise. |
| test.c:17:7:17:12 | call to signal | Reasserting handler bindings introduces a race condition on nonpersistent platforms and is redundant otherwise. |
